api/service/stagedstreamsync/syncing.go

```diff
@@ -156,67 +166,119 @@ func getBlockDbPath(beacon bool, loopID int, dbDir string) string {
 		}
 	} else {
 		if loopID >= 0 {
-			return fmt.Sprintf("%s_%d", filepath.Join(dbDir, "cache/blocks_db"), loopID)
+			return fmt.Sprintf("%s_%d_%d", filepath.Join(dbDir, "cache/blocks_db"), shardID, loopID)
```

Comment:
use utils.logger() ?

Reply:
This function returns a path in string format and it does not perform any action. Sprintf doesn't print anything, it formats the string